docs/CONTRIBUTING.md

```diff
@@ -51,13 +51,42 @@ the mailing list have a tendency to fall through the cracks.
     entries to `src/types.db`, you should talk to us early in the design
     process.
 
+### Labels
+
+PRs need to be categorized into one of three categories:
+
+*   **Feature**
+
+    The code has new behavior. Performance and documentation improvements are
+    considered features. These changes would typically go into a "feature
+    release".
+*   **Bug**
```

Comment:
```suggestion
*   **Fix**
```

Reply:
Patch releases with "Fixes" or "Bug fixes" is fine, one with "Bugs" is not. :-)

Reply:
Fixed and bot has been updated.